mern-ecommerce/controllers/user.js

```diff
@@ -125,3 +125,19 @@ exports.purchaseHistory = (req, res) => {
       res.json(orders);
     });
 };
+
+exports.deleteUser = (req, res) => {
+  
+  const { userId } = req.params;
+  User.findByIdAndRemove(userId, (err, user) => {
+    if (err || !user) {
+      return res.status(400).json({
+        error: 'User not found',
```

Comment:
The error message 'User not found' is repeated in multiple places. It would be better to define it as a constant at the top of the file or in a separate error handling module to avoid duplication and make it easier to change if needed.

Reply:
@CodeHawk can you please clarify how i can solve this problem in simple terms?

Reply:
I'd be happy to clarify. The issue is that the error message 'User not found' is repeated in multiple places in the code. To avoid duplication and make it easier to change if needed, I suggest defining it as a constant at the top of the file or in a separate error handling module. This way, if you need to change the error message in the future, you only need to update it in one place.